ui/apps/platform/src/Containers/Clusters/searchFilterConfig.ts

```diff
@@ -0,0 +1,96 @@
+import {
+    clusterIdAttribute,
+    clusterKubeVersionAttribute,
+    clusterLabelAttribute,
+    clusterNameAttribute,
+    clusterPlatformTypeAttribute,
+    clusterTypeAttribute,
+} from 'Components/CompoundSearchFilter/attributes/cluster';
+import type {
+    CompoundSearchFilterAttribute,
+    CompoundSearchFilterEntity,
+} from 'Components/CompoundSearchFilter/types';
+
+function createStatusAttribute(entity: string): CompoundSearchFilterAttribute {
+    return {
+        displayName: 'Status',
+        filterChipLabel: `${entity} status`,
+        searchTerm: `${entity} status`,
+        inputType: 'select',
+        inputProps: {
+            options: [
+                { label: 'Degraded', value: 'DEGRADED' },
+                { label: 'Healthy', value: 'HEALTHY' },
+                { label: 'Unavailable', value: 'UNAVAILABLE' },
+                { label: 'Unhealthy', value: 'UNHEALTHY' },
+                { label: 'Uninitialized', value: 'UNINITIALIZED' },
+            ],
```

Comment:
What are pro and con to factor out `options` as array at module (file) scope for reuse.

Policy criteria does something similar, and half the time I like it and other half it slows me down.

Reply:
I went back and forth on this one. It feels like it should be pulled out for reuse, but at the same time feels so specific to the search filter that I don't love the idea abstracting it.